bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -141,9 +152,25 @@ public Builder setStatsLogger(StatsLogger statsLogger) {
             return this;
         }
 
+
+        public Builder dnsResolver(DNSToSwitchMapping dnsResolver) {
+            this.dnsResolver = dnsResolver;
+            return this;
+        }
+
+        public Builder requestTimer(HashedWheelTimer requestTimer) {
```

Comment:
I forgot to tell that this class is from netty 3, netty 4 and 5 use different package names and so we will be stuck to netty 3